cmake/pkg_build/EkatBuildKokkos.cmake

```diff
@@ -58,17 +58,40 @@ macro(BuildKokkos)
 
     add_subdirectory(${Kokkos_SOURCE_DIR} ${Kokkos_BINARY_DIR})
 
-    set (Kokkos_INCLUDE_DIRS
-      ${Kokkos_SOURCE_DIR}/core/src
-      ${Kokkos_SOURCE_DIR}/algorithms/src
-      ${Kokkos_BINARY_DIR}
+    #set (Kokkos_INCLUDE_DIRS
```

Comment:
Do you anticipate that we'll need this commented code in the future?

Reply:
Ah, no, i just forgot to clean up completely. :)